models/labels/aztec_v2/aztec_v2_contract_labels.sql

```diff
@@ -0,0 +1,31 @@
+{{config(alias='aztec_v2_contract_labels')}}
+
+with contract_labels as (
+    SELECT array('ethereum') as blockchain,
+        contract_address as address, 
+        description as name,
+        contract_type as category,
+        'jackiep00' as contributor,
+        'wizard' as source,
+        date('2022-09-19') as created_at,
+        now() as updated_at,
+        version, 
+        protocol
+    from
+    (SELECT protocol, contract_type, version, description, contract_address
+    FROM (VALUES
+        ('Aztec RollupProcessor','Rollup',           '1.0',     'Prod Aztec Rollup',            '\xFF1F2B4ADb9dF6FC8eAFecDcbF96A2B351680455'::bytea),
```

Comment:
in v2 addresses are in the format '0x...' instead of '\x...'
This needs to be changed for all addresses (e.g., replace
`'\xFF1F2B4ADb9dF6FC8eAFecDcbF96A2B351680455'::bytea`
by
`'0xFF1F2B4ADb9dF6FC8eAFecDcbF96A2B351680455'`